server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/MessageRoutesExpireTest.java

```diff
@@ -0,0 +1,284 @@
+                    .get("taskId");
+
+                given()
+                    .basePath(TasksRoutes.BASE)
+                .when()
+                    .get(taskId + "/await")
+                .then()
+                    .body("type", Matchers.is("ExpireInboxesTask"))
+                    .body("status", is("completed"))
+                    .body("taskId", is(notNullValue()))
+                    .body("additionalInformation.type", is("ExpireInboxesTask"))
+                    .body("additionalInformation.runningOptions.usersPerSecond", is(RunningOptions.DEFAULT.getUsersPerSecond()))
+                    .body("additionalInformation.runningOptions.byExpiresHeader", is(true))
+                    .body("additionalInformation.runningOptions.olderThan", is(nullValue()))
+                    .body("additionalInformation.inboxesExpired", is(0))
+                    .body("additionalInformation.inboxesFailed", is(0))
+                    .body("additionalInformation.inboxesProcessed", is(0))
+                    .body("additionalInformation.messagesDeleted", is(0));
+            }
+
+            @Test
+            void expireInboxesShouldReturnTaskDetailsWhenMail() throws Exception {
+                usersRepository.addUser(USERNAME, "secret");
+                MailboxSession systemSession = mailboxManager.createSystemSession(USERNAME);
+                mailboxManager.createMailbox(INBOX, systemSession).get();
+                mailboxManager.getMailbox(INBOX, systemSession).appendMessage(
+                    MessageManager.AppendCommand.builder()
+                        .withInternalDate(new Date(System.currentTimeMillis() - 5000))
+                        .build("header: value\r\n\r\nbody"),
+                    systemSession).getId();
+
+                String taskId = when()
+                    .delete("/messages?olderThan=1s")
+                    .jsonPath()
+                    .get("taskId");
+
+                given()
+                    .basePath(TasksRoutes.BASE)
+                .when()
+                    .get(taskId + "/await")
+                .then()
+                    .body("type", Matchers.is("ExpireInboxesTask"))
+                    .body("status", is("completed"))
+                    .body("taskId", is(notNullValue()))
+                    .body("additionalInformation.type", is("ExpireInboxesTask"))
+                    .body("additionalInformation.runningOptions.usersPerSecond", is(RunningOptions.DEFAULT.getUsersPerSecond()))
+                    .body("additionalInformation.runningOptions.byExpiresHeader", is(false))
+                    .body("additionalInformation.runningOptions.olderThan", is("1s"))
+                    .body("additionalInformation.inboxesExpired", is(1))
+                    .body("additionalInformation.inboxesFailed", is(0))
+                    .body("additionalInformation.inboxesProcessed", is(1))
+                    .body("additionalInformation.messagesDeleted", is(1));
+            }
```

Comment:
Maybe adding a test to show it does not delete messages not older than the value specified in the task?